BookPlayer/Models/Playlist+CoreDataClass.swift

```diff
@@ -123,4 +126,18 @@ public class Playlist: LibraryItem {
         self.desc = "\(books.count) Files"
         self.addToBooks(NSOrderedSet(array: books))
     }
+
+    func sort(by sortType: PlayListSortOrder) {
```

Comment:
let's add a `throws` to the function, so the error can be handled where it's being called, and we avoid declaring a delegate

Reply:
👍